### PR TITLE
Add get_angle_radians function

### PR DESCRIPTION
**Pull request by pushfoo**
Added to support #1176 and simplify its code, as well as help users who are using libraries that expect radians instead of degrees.

### CODE REVIEW

File: arcade/__init__.py [hunk 1]
```diff
@@ -140,6 +140,7 @@ def configure_logging(level: int = None):
 from .geometry_generic import rotate_point
 from .geometry_generic import clamp
 from .geometry_generic import get_angle_degrees
+from .geometry_generic import get_angle_radians
```
Comment (einarf) [inline arcade/__init__.py:143]: It should also be  in `__all__` for

Comment (pushfoo) [reply arcade/__init__.py:143]: There's a comment above `__all__` that says it's generated. Is that out of date or do I need to run a script for that?

Comment (einarf) [reply arcade/__init__.py:143]: Remove that comment. It's no longer generated because it has too manu issues,

Comment (pushfoo) [reply arcade/__init__.py:143]: Done and pushed.

**Pull request merged**